package.json

```diff
@@ -147,6 +149,7 @@
     "redux": "^4.2.1",
     "redux-thunk": "^2.4.2",
     "rollup": "^3.29.3",
+    "rollup-plugin-multi-input": "^1.4.1",
```

Comment:
Is this used? If not, please delete and run `npm i` to update `package-lock.json`.

Reply:
Done!